docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
+
+Theia is the command-line tool which provides access to Theia network flow
```

Comment:
Honest question: should we refer to `theia` with lowercase initial and monospaced font?